.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+# Onboarding Feature Deployment
+
+> [!WARNING]
+> Pull requests merged to `main` will be released to production, please ensure the checklist below is complete
+
+Before any work can be merged to main in must meet the definition of done and be ready to deploy. While many of these tasks will be automated, the reviewers must take the responsibility of confirming the checklist below has been completed before this ticket can be merged.
+
+## Checklist
+
+- [ ] this PR meets the acceptance criteria of the ticket
+
+- [ ] tests have been written to cover any new or updated functionality
+
+- [ ] new configuration parameters have been deployed to all environments, see [configuration management](https://govukverify.atlassian.net/l/cp/N7q3Vh3r).
+
+- [ ] all external dependencies have been completed and resolved
+
+## Summary
+
+[ _please summarise this PR_ ]
```

Comment:
How would we feel about adding some rough headings here? I'm thinking something like:
## [Ticket]: Title 
### This PR Adds:
### This PR Modifies:
### This PR Removes:
### Notes for reviewer:

Reply:
Hmm yeah what I've described feels like a changelog with (extra? less?) steps. I'm happy either way, maybe best, for now, to leave this simple and modify it in future if we feel like it could be better